include/avif/avif.h

```diff
@@ -749,6 +749,13 @@ AVIF_API void avifImageStealPlanes(avifImage * dstImage, avifImage * srcImage, a
 // the number of threads that can ever be in flight at a given time, please account for this
 // accordingly.
 
+// ---------------------------------------------------------------------------
+// Scaling
+
+// Scales the YUV/A planes in-place. dstWidth and dstHeight must both be <= AVIF_DEFAULT_IMAGE_DIMENSION_LIMIT and
+// dstWidth*dstHeight should be <= AVIF_DEFAULT_IMAGE_SIZE_LIMIT. diag is optional and can be NULL.
```

Comment:
Remove "diag is optional and can be NULL" for now. Please see my comment on this issue in src/scale.c.

Reply:
Done.